membership-attribute-service/app/services/mail/Emails.scala

```diff
@@ -21,15 +21,15 @@ object Emails {
         "first_name" -> contact.firstName.getOrElse(""),
         "last_name" -> contact.lastName,
         "payment_method" -> paymentMethod.valueForEmail,
-        "product_type" -> plan.productType,
+        "product_type" -> productType.productTypeString,
```

Comment:
productType is now a value class rather than a string so it needs unpacking when it's serialised